source/contributing/spongedocs.rst

```diff
@@ -52,6 +52,17 @@ writing Sponge Documentation. This list may get added to (or bent out of shape)
      Contributors are discouraged from commenting in code blocks wherever possible. Simple place-holder text may be
      necessary in some examples. Ideally, code block examples will be short, and followed by an explanation for each
      example in the body text. Of course, there may be some concepts that cannot be illustrated with a short example.
+  ii. Use the following code block types for particular code blocks:
+
+     * Groovy/Gradle-Files -> ``groovy``
+     * Hocon-Config -> ``guess`` (hocon highlighting is not yet supported)
```

Comment:
Not sure on this one: `guess` will hardly find the right one. `groovy` yields good results but is logically wrong. `js`/´json` fail on comments.

Reply:
Ref: https://github.com/SpongePowered/SpongeDocs/issues/713 Add code highlighting support for hocon examples